src/blocks/scratch3_sensing.js

```diff
@@ -23,6 +24,8 @@ class Scratch3SensingBlocks {
         this.runtime.on('ANSWER', this._onAnswer.bind(this));
         this.runtime.on('PROJECT_START', this._resetAnswer.bind(this));
         this.runtime.on('PROJECT_STOP_ALL', this._clearAllQuestions.bind(this));
+
+        this.timer = new Timer();
```

Comment:
A few things here:
1. Can you put this up with the other properties that are initialized (below _answer ^)
2. Can you change this to be `_` prefixed if it isn't going to be accessed externally.
3. There are two other private properties you added in `getLoudness`, (`this.loudness` and `this.loudnessTime`), those should also be initialized here, use the `_` prefix, and can you update the names to be more clear (`this.loudness` should probably be `this._cachedLoudness` and `this.loudnessTime` should probably be `this._loudnessLastMeasuredTime` or something similarly explicit).

Reply:
all set